test_elasticsearch/test_server/test_helpers.py

```diff
@@ -16,7 +16,7 @@ def bulk(self, *args, **kwargs):
         self._called += 1
         if self._called in self._fail_at:
             raise self._fail_with
-        return  self.client.bulk(*args, **kwargs)
+        return self.client.bulk(*args, **kwargs)
```

Comment:
please configure your editor not to make changes like these. It makes the PR hard to review when they are unrelated stylistic changes in unrelated places.